packages/cli/src/services/public-api-key.service.ts

```diff
@@ -0,0 +1,84 @@
+import { randomBytes } from 'crypto';
```

Comment:
Could use `node:` prefix to be explicit this is a node native lib

```suggestion
import { randomBytes } from 'node:crypto';
```